datascience/tables.py

```diff
@@ -2655,7 +2655,42 @@ def to_csv(self, filename):
         self.to_df().to_csv(filename, index=False)
 
     def to_array(self):
-        """Convert the table to a structured NumPy array."""
+        """Convert the table to a structured NumPy array.
+
+        The elements in the resulting array preserve row and column replacement
```

Comment:
I did not  understand what "row and column replacement" means.  Actually, I'm having a hard time understanding this entire paragraph.  Can we reword?  Maybe the result is an array of rows, where each row is represented as a sequence of fields, and each field corresponds to a column label?  Or something?

Reply:
I meant to just type "row and column placement", but I just pushed a commit which the description is much simpler.